components/layout_2020/taffy/layout.rs

```diff
@@ -108,7 +108,7 @@ impl taffy::LayoutPartialTree for TaffyContainerContext<'_> {
         Self: 'a;
 
     fn get_core_container_style(&self, _node_id: taffy::NodeId) -> Self::CoreContainerStyle<'_> {
-        TaffyStyloStyle(self.style)
+        TaffyStyloStyle::new(self.style, false)
```

Comment:
Bool arguments aren't very clear by themselves

```suggestion
        TaffyStyloStyle::new(self.style, false /* is_replaced */)
```